src/EmailRecoveryManager.sol

```diff
@@ -441,17 +441,27 @@ abstract contract EmailRecoveryManager is
     }
 
     /**
-     * @notice Removes all state related to an account.
+     * @notice Removes all state related to msg.sender.
      * @dev In order to prevent unexpected behaviour when reinstalling account modules, the module
      * should be deinitialized. This should include remove state accociated with an account.
      */
     function deInitRecoveryModule() internal onlyWhenNotRecovering {
-        delete recoveryConfigs[msg.sender];
-        delete recoveryRequests[msg.sender];
+        address account = msg.sender;
+        deInitRecoveryModule(account);
+    }
+
+    /**
+     * @notice Removes all state related to an account.
+     * @dev In order to prevent unexpected behaviour when reinstalling account modules, the module
+     * should be deinitialized. This should include remove state accociated with an account.
+     */
+    function deInitRecoveryModule(address account) internal onlyWhenNotRecovering {
```

Comment:
For completeness even though risk is low, can you add unit tests for this new function?

You can just replicate `EmailRecoveryManager_deInitRecoveryModule_Test`, but for the new internal function with the account argument. You could even add tests in the same file.

Would need to update the harness to expose the internal function